packages/vats/src/bootstrap.js

```diff
@@ -271,11 +320,11 @@ export function buildRootObject(vatPowers, vatParameters) {
         issuer: centralIssuer,
         brand: centralBrand,
         bankDenom: CENTRAL_DENOM_NAME,
-        bankPayment: centralBootstrapPayment,
+        bankPayment: bankBootstrapPayment,
       }),
       // We still create demo currencies, but not obviously fake ones unless
       // $FAKE_CURRENCIES is given.
-      ...demoIssuerEntries(noFakeCurrencies),
+      ...demoIssuers,
```

Comment:
update / delete the comment with this change?

Reply:
Yes, what a tangle. I suppose #4021 represents at least some of the work we should do in this area.